CHANGES.txt

```diff
@@ -1,9 +1,19 @@
 Version 2.0.0 (unreleased)
 ==========================
 
-* **Breaking**: New account currencies now default to ``DEFAULT_CURRENCY`` rather than all available currencies (trading accounts should be the only accounts with multiple currencies).
+**Important:** It is strongly recommended that you
+**upgrade to Hordak 1.16 before upgrading to Hordak 2.0**. Update your code
+to address any depreciation warnings in 1.16 before upgrading to Hordak 2.0.
+
+* **Breaking**: ``to_account()`` was deprecated in 1.16. This old implementation has now been removed
```

Comment:
I think you mean `transfer_to()` and `accounting_transfer_to()`

Reply:
Ah yes, thank you @nitsujri! Fixed.